sql/core/src/test/java/test/org/apache/spark/sql/JavaBeanWithArraySuite.java

```diff
@@ -0,0 +1,195 @@
+package test.org.apache.spark.sql;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Encoder;
+import org.apache.spark.sql.Encoders;
+import org.apache.spark.sql.test.TestSparkSession;
+import org.apache.spark.sql.types.*;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
```

Comment:
The import orders here are not compliant with Spark codebase. You can follow the style in other tests like `JavaApplySchemaSuite`.